app/core/users/endpoints_users.py

```diff
@@ -826,6 +826,39 @@ async def update_current_user(
     await cruds_users.update_user(db=db, user_id=user.id, user_update=user_update)
 
 
+@router.patch(
```

Comment:
I think a POST may be more adapted as this start an action, and can not be called twice